pipeline-modules/app-service/gcp/docker-build.yaml

```diff
@@ -0,0 +1,37 @@
+provisioner: gcp
+name: docker-build
+version: 1
+revision: 1
+displayName: Docker Build
+description: This plugin only builds the Docker image, useful for running checks
+target: ""
+category: containerise
+keywords:
+  - Docker
+  - Build
+author: CloudCover
+meta: {}
+inputs:
+  properties:
+    docker_build_args:
+      title: Docker Build Arguments
+      description: Arguments for the docker build command
+      type: array
+      default: []
+      items:
+        type: string
+    working_dir:
+      title: Working directory
+      description: The current working directory to execute command
+      type: string
+      default: repo
+template: |
+  steps:
+  - id: 'Docker Build'
+    name: 'gcr.io/cloud-builders/docker'
+    entrypoint: 'bash'
```

Comment:
if you are just going to run a docker command, you don't need to override the endpoint or use "docker" in the args:
https://github.com/GoogleCloudPlatform/cloud-builders/tree/master/docker

Reply:
I've made the change as requested, please check